source/tyr/tyr/tasks.py

```diff
@@ -297,6 +297,28 @@ def purge_instance(instance_id, nb_to_keep):
     for path in to_remove:
         shutil.rmtree(path)
 
+@celery.task()
+def purge_jobs():
+    """
+    Delete old jobs in database and backup folders associated
+    """
+    instances = models.Instance.query_existing().all()
+    time_limit = datetime.utcnow() - timedelta(days=current_app.config['JOB_MAX_PERIOD_TO_KEEP'])
+    logger = logging.getLogger(__name__)
+    logger.info('Purge old jobs and datasets backup created before {}'.format(time_limit))
+
+    for instance in instances:
+        datasets_to_delete = instance.delete_old_jobs_and_list_datasets(time_limit)
+
+        # instance_config = load_instance_config(instance.name)
```

Comment:
Why is that commented?

Reply:
because it isn't needed and I was forgotten here... Will delete it